# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/cairo/cairo/src/cairo-path-fixed-private.h
# Commit: 3cdfb39376b7
# Full Hash: 3cdfb39376b736597fe658a322389117686b7e82
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2024-05-07 09:13:07
# Regressor Bug: 1892913
# File Overlap Count: 3
# Description:
#   Bug 1892913 - patch 1 - Update cairo source to 1.18.0 r=gfx-reviewers,lsalzman
#   
#   Wholesale replacement of files in gfx/cairo/cairo with contents of the 1.18.0 release tarball,
#   omitting subdirectories that are not part of the core library build.
#   
# ==============================================================================

diff -r 9daa351f875b -r 3cdfb39376b7 gfx/cairo/cairo/src/cairo-path-fixed-private.h
--- a/gfx/cairo/cairo/src/cairo-path-fixed-private.h	Mon May 06 19:22:35 2024 +0000
+++ b/gfx/cairo/cairo/src/cairo-path-fixed-private.h	Mon May 06 19:25:12 2024 +0000
@@ -126,7 +126,7 @@
 			  cairo_fixed_t			     tx,
 			  cairo_fixed_t			     ty);
 
-cairo_private unsigned long
+cairo_private uintptr_t
 _cairo_path_fixed_hash (const cairo_path_fixed_t *path);
 
 cairo_private unsigned long